hooks/open-telemetry/README.md

```diff
@@ -1,38 +1,92 @@
 # OpenTelemetry Hook
 
-### Requirements
+## Requirements
 
 - open-feature/go-sdk >= v1.3.0
 
 ## Usage
 
-For this hook to function correctly a global `TracerProvider` must be set, an example of how to do this can be found below.
+## Metric hook
 
-The `open telemetry hook` taps into the after and error methods of the hook lifecycle to write `events` and `attributes` to an existing `span`.
+This hook performs metric collection by tapping into various hook stages. Given below are the metrics are extracted 
```

Comment:
Please do a line break at a new sentence instead of after a character count (where possible).